param/parameterized.py

```diff
@@ -1137,6 +1138,48 @@ def watch(self_,parameter_name,parameter_attribute=None,fn=None):
             subscribers[parameter_attribute].append(fn)
 
 
+    # TODO:
+    #
+    #   * error if not watched? (would then need a way to ask about status)
```

Comment:
Probably should error yes.

Reply:
I guess you can catch the exception if you want to unwatch something not knowing whether you've unwatched it already.